modules/common/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/utils/IgluUtils.scala

```diff
@@ -310,16 +284,8 @@ object IgluUtils {
     validationInfoContexts: List[SelfDescribingData[Json]]
   )
 
-  /** Build `BadRow.SchemaViolations` from a list of `FailureDetails.SchemaViolation`s */
-  def buildSchemaViolationsBadRow(
-    vs: NonEmptyList[FailureDetails.SchemaViolation],
-    pee: Payload.PartiallyEnrichedEvent,
-    re: Payload.RawEvent,
-    processor: Processor
-  ): BadRow.SchemaViolations =
-    BadRow.SchemaViolations(
-      processor,
-      Failure.SchemaViolations(Instant.now(), vs),
-      Payload.EnrichmentPayload(pee, re)
-    )
+  private implicit class ExtraEitherTSyntax[F[_], A, B](val eitherT: EitherT[F, A, B]) extends AnyVal {
+    def toIorT(implicit F: Monad[F]): IorT[F, A, B] =
+      IorT(eitherT.value.map(_.toIor))
+  }
```

Comment:
I think you don't need this, although I know copied it from me!  EitherT [already has a method](https://github.com/typelevel/cats/blob/v2.10.0/core/src/main/scala/cats/data/EitherT.scala#L707) called `toIor`.  I missed it earlier because I was expecting it to be called `toIorT`.